shared/src/packages/users/libs/enums/user-validation-message.enum.ts

```diff
@@ -1,6 +1,8 @@
 const UserValidationMessage = {
   EMAIL_REQUIRED: 'Email is required',
   EMAIL_WRONG: 'Email is wrong',
+  NAME_REQUIRED: 'Name is required',
+  PASSWORD_REQUIRED: 'Password is required',
 } as const;
```

Comment:
A dot should be added at the end of each message

Reply:
done